### PR TITLE
Use official width/height

### PR DESCRIPTION
**Pull request by apple502j**
@Kenny2github Review?

### CODE REVIEW

File: EmbedScratch.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ function renderEmbedScratch ($input, $argv, $parser) {
 				. $height / $width * 100
 				. "px\">"
 				. "<iframe "
-				. "allowtransparency=\"false\" "
+				. "allowtransparency=\"true\" "
```
Comment (Kenny2github) [inline EmbedScratch.php:69]: Why?

Comment (apple502j) [reply EmbedScratch.php:69]: Official embed code has that.

**Comment by Kenny2github** [comment]
Also what do you mean by "official height"?

**Comment by apple502j** [comment]
```html
<iframe src="https://scratch.mit.edu/projects/303846468/embed" allowtransparency="true" width="485" height="402" frameborder="0" scrolling="no" allowfullscreen></iframe>
```
Example of "official" embed code.

**Comment by Kenny2github** [comment]
Oh, so this matches the code that Scratch gives you?

**Comment by apple502j** [comment]
@Kenny2github Basically.

**Review by Kenny2github — APPROVED** [review]
LGTM

**Pull request merged**